valve/gamemodes/llhl.cfg

```diff
@@ -0,0 +1,117 @@
+//Llhl
+//Liga Latinoamericana de Half-Life
+
+sv_ag_gametype			"llhl"
+sv_ag_pure 			    0
+
+mp_teamlist				"blue;red"
+mp_teamplay				1
+mp_friendlyfire			1
+mp_forcerespawn			1
+mp_weaponstay			0
+mp_timelimit			20
+mp_fraglimit			0
+mp_falldamage			0
+mp_flashlight           0
+mp_respawn_fix          1
+
+sv_ag_start_minplayers  2
+sv_ag_vote_failed_time  30
+sv_ag_vote_setting      1
+sv_ag_max_spectators    16
+sv_ag_vote_kick         1
+sv_ag_allow_vote        1
+sv_ag_vote_map          1
+sv_ag_vote_start        1
+ag_rpg_fix              1
+ag_gauss_fix            1
+
+sv_ag_lj_timer			0
+sv_ag_wallgauss			1
+sv_ag_headshot			3
+sv_ag_blastradius		1
+sv_ag_oldphysics		1
+sv_ag_spawn_volume		1
+
+sv_ag_ban_crowbar		0
+sv_ag_ban_glock			0
+sv_ag_ban_357			0
+sv_ag_ban_mp5			0
+sv_ag_ban_shotgun		0
+sv_ag_ban_crossbow		0
+sv_ag_ban_rpg			0
+sv_ag_ban_gauss			0
+sv_ag_ban_egon			0
+sv_ag_ban_hornet		0
+sv_ag_ban_hgrenade		0
+sv_ag_ban_satchel		0
+sv_ag_ban_tripmine		0
+sv_ag_ban_snark			0
+
+sv_ag_ban_m203			0
+sv_ag_ban_9mmar			0
+sv_ag_ban_bockshot		0
+sv_ag_ban_uranium		0
+sv_ag_ban_bolts			0
+sv_ag_ban_rockets		0
+sv_ag_ban_357ammo       0
+
+sv_ag_ban_health		0
+sv_ag_ban_armour		0
+sv_ag_ban_recharg		0
+sv_ag_ban_longjump		0
+
+sv_ag_start_crowbar		1
+sv_ag_start_glock		1
+sv_ag_start_357			0
+sv_ag_start_mp5			0
+sv_ag_start_shotgun		0
+sv_ag_start_crossbow	0
+sv_ag_start_rpg			0
+sv_ag_start_gauss		0
+sv_ag_start_egon		0
+sv_ag_start_hornet		0
+sv_ag_start_hgrenade	0
+sv_ag_start_satchel		0
+sv_ag_start_tripmine	0
+sv_ag_start_snark		0
+
+sv_ag_start_m203		0
+sv_ag_start_9mmar		68
+sv_ag_start_bockshot	0
+sv_ag_start_uranium		0
+sv_ag_start_bolts		0
+sv_ag_start_rockets		0
+sv_ag_start_357ammo		0
+
+sv_ag_start_health		100
+sv_ag_start_armour		0
+sv_ag_start_longjump	0
+
+sv_ag_dmg_crowbar           30
+sv_ag_dmg_glock             12
+sv_ag_dmg_357               40
+sv_ag_dmg_mp5               12
+sv_ag_dmg_shotgun			20
+sv_ag_dmg_bolts_normal		120
+sv_ag_dmg_bolts_explosion   50
```

Comment:
Value is wrong, it should be 40. Please check out tdm.cfg for default values for damage to ensure values are right, as fars I know EHLL only decreases egon damage right?

Reply:
Fixed. EHLL increases crowbar and hornet damage and decreases egon damage.